### PR TITLE
Debug postgres container

### PR DESCRIPTION
**Pull request by nahguam**
# stream-registry PR

Debug postgres container

# PR Checklist Forms

- [ ] CHANGELOG.md updated
- [ ] Reviewer assigned
- [ ] PR assigned (presumably to submitter)
- [ ] Labels added (enhancement, bug, documentation)

### CODE REVIEW

File: it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/StreamRegistryIT.java [hunk 1]
```diff
@@ -46,36 +43,34 @@
...
-          .withFixedExposedPort(POSTGRES_PORT, 5432);
+          .withEnv("POSTGRES_DB", "streamregistry");
 
   @BeforeClass
   public static void before() {
     String[] args = new String[]{
         "--server.port=0",
         "--schema.registry.url=http://schema-registry",
-        "--spring.datasource.url=jdbc:postgresql://localhost:" + POSTGRES_PORT + "/streamregistry",
+        "--spring.datasource.url=jdbc:postgresql://localhost:" + postgres.getMappedPort(5432) + "/streamregistry",
```
Comment (kyrsideris) [inline it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/StreamRegistryIT.java]: Does this port get mapped into host's ports? Consider randomizing it!

Comment (nahguam) [reply it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/StreamRegistryIT.java]: I was manually randomizing previously, but I discovered testcontainers already does it for you. `getMappedPort` returns you the random port that it assigned to the host port.

Comment (bsideup) [reply it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/StreamRegistryIT.java]: it is also **highly** recommended to not use `localhost` here, see:
https://www.testcontainers.org/features/networking/

there is `container.getContainerIpAddress()` for that.

Comment (bsideup) [reply it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/StreamRegistryIT.java]: If you use [the PostgreSQL module](https://www.testcontainers.org/modules/databases/postgres/), you can even replace it with `container.getJdbcUrl()`

**Pull request merged**